packages/promptable/tsconfig.json

```diff
@@ -37,8 +37,8 @@
             ],
             "@loaders/*": [
                 "src/loaders/*"
-            ],
-        },
```

Comment:
This file was invalid json, which would fail whenever I pulled it into the jest configuration.